### PR TITLE
List style bullet points

### PR DESCRIPTION
**Pull request by EmmanuelDrouin**
Update css of list elements with bullet points

### CODE REVIEW

**Comment by antho1404** [comment]
Deploy preview for *mesg* ready!

Built with commit 7d56490d9574a3b24d795e5b36d2c94355c6831b

https://deploy-preview-190--mesg.netlify.com

File: assets/style.scss [hunk 1]
```diff
@@ -110,23 +110,20 @@ p {
...
-  display: inline-block;
-}
-.list p {
-  margin-left:22px;
+  li {
+    font-size: 17px;
+    font-weight: normal;
+    color: var(--text-color);
+    margin-left: 1em;
+    &:before {
```
Comment (antho1404) [inline assets/style.scss]: why keep using the `:before` and not the `list-style` like we discussed?

Comment (EmmanuelDrouin) [reply assets/style.scss]: Changed my mind and switched for list-style

**Comment by EmmanuelDrouin** [comment]
@antho1404 can you review this pr?

**Comment by antho1404** [comment]
Is it normal to have lines on the menu?

Screen Shot 2019-11-29 at 16 59 58

**Comment by EmmanuelDrouin** [comment]
@antho1404 yes i updated a bit the design of the submenu to add those lines. I already plan to redesign completely the menu later when we gonna have the new organization of the website, but until this happen, The menu is better like this, especially on responsive.

**Pull request merged**